server/command.go

```diff
@@ -13,32 +13,41 @@ const (
 	responseUsername = "Matterpoll"
 
 	// Parameter: Trigger
-	commandHelpTextFormat = "To create a poll with the answer options \"Yes\" and \"No\" type `/%s \"Question\"`.\nYou can customise the options by typing `/%s \"Question\" \"Answer 1\" \"Answer 2\" \"Answer 3\"`"
-	// Parameter: Trigger, Trigger
-	commandInputErrorFormat = "Invalid Input. Try `/%s \"Question\"` or `/%s \"Question\" \"Answer 1\" \"Answer 2\" \"Answer 3\"`"
+	commandHelpTextFormat = "To create a poll with the answer options \"Yes\" and \"No\" type `/%[1]s \"Question\"`.\n" +
+		"You can customise the options by typing `/%[1]s \"Question\" \"Answer 1\" \"Answer 2\" \"Answer 3\"`\n" +
+		"Poll Settings provider further customisation, e.g. `/%[1]s \"Question\" \"Answer 1\" \"Answer 2\" \"Answer 3\" --progress --secret`. The available Poll Settings are:\n" +
+		"- `--secret`: Don't show who voted for what at the end\n" +
+		"- `--progress`: During the poll, show how many votes each answer option got\n"
+
+	// Parameter: Trigger
+	commandInputErrorFormat = "Invalid input. Try `/%[1]s \"Question\"` or `/%[1]s \"Question\" \"Answer 1\" \"Answer 2\" \"Answer 3\"`"
```

Comment:
I din't know this working until now, but it allows us a cleaner code. See https://godoc.org/fmt#hdr-Printing